### PR TITLE
feat: message to request scenes info around a parcel

### PR DESCRIPTION
**Pull request by pbosio**
# What?
Added message from renderer to kernel to request scenes info around a parcel

# Why? 
We need to fetch the info of scenes around a specific parcel. Currently we can only request info around the parcel where the player is on and with a fixed amount of scenes around.

### CODE REVIEW

File: kernel/packages/shared/atlas/sagas.ts [hunk 1]
```diff
@@ -107,32 +117,51 @@ export function* checkAndReportAround() {
...
+  yield call(reportScenes, atlasState, targets)
+}
+
+export function* reportScenesAroundParcel(action: ReportScenesAroundParcel) {
+  let atlasState = (yield select(state => state.atlas)) as AtlasState
+  while (!atlasState.marketName['0,0']) {
+    yield take(MARKET_DATA)
+    atlasState = yield select(state => state.atlas)
+  }
+
```
Comment (moliva) [inline kernel/packages/shared/atlas/sagas.ts:143]: Could we use the selector `getNameFromAtlasState` from `shared/atlas/selectors.ts` maybe? If not, please refactor to some `isMarketDataInitialized` selector.

File: kernel/packages/shared/atlas/sagas.ts [hunk 2]
```diff
@@ -107,32 +117,51 @@ export function* checkAndReportAround() {
...
+  const targets = getScenesAround(action.payload.parcelCoord, action.payload.scenesAround, atlasState)
+  yield call(reportScenes, atlasState, targets)
+}
+
+function getScenesAround(parcelCoords: { x: number; y: number }, maxScenesAround: number, atlasState: AtlasState) {
```
Comment (moliva) [inline kernel/packages/shared/atlas/sagas.ts]: Let's extract the parcelCoords type somewhere to be reused everywhere needed.

**Review by moliva — APPROVED** [review]
LGTM 👍

**Review by pablitar — APPROVED** [review]
Looks great!

**Pull request merged**